src/ARCtrl/ARC.fs

```diff
@@ -71,6 +71,25 @@ type ARC(?isa : ISA.ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSyste
 
     member this.FileSystem 
         with get() = _fs
+        and set(fs) = _fs <- fs
+
+    member this.RemoveAssay(assayIdentifier: string) =
```

Comment:
These functions should contain update contracts for all registering objects (study,investigation)

Additional helpers:
- ContractCreator Type extensions for ArcInvestigation, ArcStudy and ArcAssay in Contract namespace